.github/workflows/manual_corp_user_group_sync.yml

```
@@ -0,0 +1,31 @@
+name: Manually Sync Eras Corporate User Group Sync
+
+on:
+  workflow_dispatch:
+
+jobs:
+  manual_sync:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3.5.2
+
+    - name: Login to GitHub Container Registry
+      uses: docker/login-action@v2
+      with:
+        registry: ghcr.io
+        username: ${{ github.actor }}
+        password: ${{ secrets.GITHUB_TOKEN }}
+
+    - uses: azure/login@v1
```

this needs to be v2 to avoid node 16 deprecation warnings

```suggestion
    - uses: azure/login@v2
```